Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -242,6 +242,8 @@ function vtkOpenGLRenderWindow(publicAPI, model) {
     return model.size;
   };
 
+  publicAPI.getDevicePixelRatio = () => window.devicePixelRatio;
```

Comment:
how about having it in `RenderWindowViewNode` instead ?
It would be a simply `model` variable with `get/set` accessors but initialized to `window.devicePixelRatio || 1` in `RenderWindowViewNode.extend()`

Reply:
One wrinkle, window.devicePixelRatio can change after initializing.

Reply:
Interesting... 
that means we should probably also observe the change and propagate it by invoking a new type of event ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio#monitoring_screen_resolution_or_zoom_level_changes